src/app/icd/BUILD.gn

```diff
@@ -57,6 +57,7 @@ source_set("cluster") {
   ]
 
   public_deps = [
+    ":notifier",
```

Comment:
Is this really a public dep?  We seem to only include it in implementation, not in headers.